website/content/docs/intro/use-cases/dynamic-config-vault-dynamic-secrets.mdx

```diff
@@ -0,0 +1,269 @@
+---
+layout: docs
+page_title: Dynamic App Config using Vault Dynamic Secrets
+description: |-
+  Waypoint enables you to source config dynamically from various external data sources using config sourcer plugins - this use case depicts doing just that with Vault dynamic database secrets.
+---
+
+# Dynamic Secrets for Waypoint with Vault
+
+Hashicorp Waypoint is an application deployment platform that simplifies how users
+test and release apps onto platforms and utilize infrastructure managed by Terraform.
+After deploying your app, Waypoint can also be used to configure it! This can be
+done with static application configuration, where static environment variables
+or files can be applied to your running deployment. Alternatively, configuration
+can be sourced dynamically from external sources, using a config sourcer plugin.
+
+The [Hashicorp Vault plugin for Waypoint](https://developer.hashicorp.com/waypoint/plugins/vault)
+is one of the [config sourcer plugins](https://developer.hashicorp.com/waypoint/docs/app-config/dynamic)
+that is built-in to the Waypoint binary. This plugin can be used to source static
+values from the Vault KV secrets engine, but also for dynamic secrets! A [dynamic secret](https://developer.hashicorp.com/vault/tutorials/getting-started/getting-started-dynamic-secrets)
+is one that is managed by Vault and has a lease associated with it, where
+the secret will expire after a set time to live (TTL). This is a huge benefit to
+the security of your application, as any possibly leaked secrets would automatically
+expire after a pre-configured amount of time, among other things including that
+developers and operators don’t need to know secrets for their deployed applications.
+
+Prior to using Waypoint to deploy my Nomad jobs, I’ve relied on [Nomad’s integration
```

Comment:
A quick note on this paragraph, since this isn't a blog post with an Author attached to it, the voice here should be less "I" "My" because the reader won't really know who that is.

For this example specifically, you could rephrase it to say like

> Prior to Waypoint, Nomad users would have to rely on Nomads integration with Vault....

But this applies to the rest of this post too!

Reply:
Makes sense. Since this section is new to the site, _"I"_ wasn't sure which way to go with it, but I'll undo the 1st person.